HTML: the condition for opening a popup by window.open, and BarProp values for each case

arai-a opened the pull request:
For https://github.com/whatwg/html/issues/5872 and https://github.com/whatwg/html/issues/4431

annevk commented on html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/resources/is-popup-barprop.html:
```diff
@@ -0,0 +1,15 @@
+<script src="/common/PrefixedPostMessage.js"></script>
```
This is neat, didn't know about this.

annevk reviewed (commented):
Found a minor issue while running these locally.

arai-a commented on html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/open-features-is-popup-condition.html:
```diff
@@ -0,0 +1,91 @@
+
+<!-- user agents are not required to support open features other than `noopener`
+     and on some platforms position and size features don't make sense -->
+<meta name="flags" content="may" />n
```
thanks!

arai-a committed:
HTML: the condition for opening a popup by window.open, and BarProp values for each case

For https://github.com/whatwg/html/issues/5872 and https://github.com/whatwg/html/issues/4431

arai-a committed:
Remove unnecessary flag
